instrumentation/jsf/myfaces-1.2/javaagent/myfaces-1.2-javaagent.gradle

```diff
@@ -0,0 +1,27 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.apache.myfaces.core"
+    module = "myfaces-impl"
+    versions = "[1.2,3)"
+    extraDependency "jakarta.el:jakarta.el-api:3.0.3"
+    assertInverse = true
+  }
+}
+
+dependencies {
+  library group: 'org.apache.myfaces.core', name: 'myfaces-impl', version: '2.3.7'
```

Comment:
Right now we have a myfaces-1.2 module that tests version 2+ and a separate 1.2 testing module. Wouldn't it make more sense to keep 1.2 testing in the myfaces-1.2 module and have a separate one for 2+ tests?

Also, have you considered using a separate test task for 2+ tests instead of a separate module? Like `jms2Test` in the JMS instrumentation.

Reply:
Yeah, this turned out weird. I initially wanted to add integration with only 2.2 and 2.3 but as it turned out to also work for 1.2 I changed the version number to 1.2, but i still wanted to use latestDepTestLibrary to test latest version from 2.x series. As 1.2 & 2.2 have a bit different dependencies I suspect that putting them in one module will make getting them to run with correct dependencies tricky.